hw4/index.html

```diff
@@ -1,12 +0,0 @@
-<!DOCTYPE html>
```

Comment:
There is still changes from the other homework, that shouldn't be there. It you have questions about how to create a new branch from the main one, you can write me and we'll make it together. But you are doing this mistakes 3 times in a row, despite the fact I write about in each time. What the problem with this moment?